framework/components/ecl-files/ecl-files.twig

```diff
@@ -1,112 +1,147 @@
-<!-- Default -->
-<div class="file">
-  <span class="icon icon--file file__icon"></span>
-  <div class="file__metadata">
-    <span class="file__title">File title example</span>
-    <div class="file__info">
-      <span class="file__content-language">English</span> (213.25 kB - PDF - 4 pages)
+{% if variant is defined %}
+
+  {% if variant == "translations" %}
+
+<!-- Translations -->
+<div class="ecl-file">
```

Comment:
well I kept a similar markup between all modifiers.
I could set is as a section, but should we also use section for the other ones?